src/Faker/Provider/Image.php

```diff
@@ -90,7 +108,8 @@ public static function image(
         $fullPath = true,
         $randomize = true,
         $word = null,
-        $gray = false
+        $gray = false,
+        $format = 'png'
```

Comment:
```suggestion
        $format = 'png',
```

Reply:
This is only php 8+ @bram-pkg

Reply:
Oh right, ugh.

Reply:
Is PHP 8+ not supported? I used named arguments in the unit tests.

Reply:
> Is PHP 8+ not supported? I used named arguments in the unit tests.

No it is 7.1+ see pipelines